.gitignore

```diff
@@ -21,3 +21,7 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+
+
+/.idea
```

Comment:
Looks like the .idea directory is still getting tracked by git. See https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore on how to untrack this directory